bridge_adaptivity/module/templates/module/modals/delete_object_modal.html

```diff
@@ -0,0 +1,30 @@
+{% load bootstrap3 %}
```

Comment:
ditto

Reply:
left only one of them - `delete_object.html`